src/lib/mina_block/body_reference.ml

```diff
@@ -10,19 +10,42 @@ open Core_kernel
 
 [%%versioned
 module Stable = struct
-  module V1 = struct
-    (* type t = State_body_hash.Stable.V1.t *)
-    type t = unit [@@deriving compare, sexp, yojson]
+  module V2 = struct
```

Comment:
Seemingly, there is no reason here to introduce a new V2 type. We should just clobber this type and keep all the `Mina_block.*` types at V1.

Reply:
Well, there is still compatible where it's `unit`

Reply:
Ok, got it.